platform.js

```diff
@@ -415,12 +416,12 @@ module.exports = {
         });
       }
       else {
-        var lnxCommand = "df --block-size=K --output=avail " + module.exports.getInstallDir();
+        var lnxCommand = "df -k " + dir + " | awk 'NR==2 {print $4}'";
 
```

Comment:
@tejohnso because the other one is not unix compatible. It only work on linux and crashes on our dev mac os env. cheers